log_outgoing_requests/models.py

```diff
@@ -99,3 +117,35 @@ def url_parsed(self):
     @property
     def query_params(self):
         return self.url_parsed.query
+
+
+class OutgoingRequestsLogConfig(SingletonModel):
+    class SaveLogsChoices(models.IntegerChoices):
+        NO = 0, _("No")
+        YES = 1, _("Yes")
+
+        __empty__ = _("Use default")
+
+    save_to_db = models.IntegerField(
+        _("Save logs to database"),
+        choices=SaveLogsChoices.choices,
+        blank=True,
+        null=True,
+        help_text=_(
+            "Whether request logs should be saved to the database (default: {default})"
+        ).format(default=settings.LOG_OUTGOING_REQUESTS_DB_SAVE),
+    )
+    save_body = models.IntegerField(
+        _("Save request + response body"),
+        choices=SaveLogsChoices.choices,
+        blank=True,
+        null=True,
+        help_text=_(
+            "Wheter the body of the request and response should be logged (default: "
+            "{default}). This option is ignored if 'Save Logs to database' is set to "
+            "False."
+        ).format(default=settings.LOG_OUTGOING_REQUESTS_SAVE_BODY),
+    )
+
+    class Meta:
+        verbose_name = _("Outgoing Requests Logs Configuration")
```

Comment:
```suggestion
        verbose_name = _("outgoing request logs configuration")
```
Django takes care of capitalization, so use lower case for `verbose_name` (this also applies for model fields)